tor_core/config.py

```diff
@@ -156,6 +156,9 @@ class Config:
 
 # ----- Compatibility -----
 config = Config
+config.name = None
+config.bot_version = 0  # this should get overwritten by the bot process
```

Comment:
Good use of commenting here.

For bot versions, though, are we trying to limit them to integers? If not, having the `0` as a string might work better.